Week3/hw/app4.js

```diff
@@ -0,0 +1,9 @@
+//  4. Create a function named `vehicleType` that receives a color, and a code, 1 for car, 2 for motorbike. And prints _'a blue motorbike'_ for example when called as `vehicleType("blue", 2)`
+
+function vehicleType(color, code) {
+  if (('blue', 2)) {
```

Comment:
Can you take a look at this again? I don't think this is correct (I don't think that this compiles..)